### DISCUSSION TITLE
The URL of the Google Form for the MVC Submission is invalid.

### DISCUSSION THREAD

**Discussion by david54573** [DLR-VSDC/IEEE-MVC-2023#40] [Q&A] [answered]
As the title suggests, I cannot fill out the URL to your Google Form.

**Comment by JBRDLR** [comment]
@rpintodecastro please provide a new link for this page:
https://github.com/DLR-VSDC/IEEE-MVC-2023/blob/main/SUBGUIDE.md
thx 
Jonathan

**Comment by rpintodecastro** [comment, marked as answer]
@JBRDLR ; @david54573 
Try this link:
https://docs.google.com/forms/d/e/1FAIpQLSf0y_SD_n7wPls6xExOJOcU9CL_4K4JdZt13a3Mx1kzHPUOpw/viewform?usp=sf_link

**Reply by david54573** [reply]
Thank you, I have completed filling out.
